microscopium/serve.py

```diff
@@ -267,6 +270,19 @@ def update_table(indices, df, table):
     filtered_df = df.iloc[indices]
     table.source.data = ColumnDataSource(filtered_df).data
 
+def switch_modes_button_group():
+    from bokeh.models.widgets import RadioButtonGroup
+
+    radio_button_group = RadioButtonGroup(
+        labels=["tSNE", "UMAP", "PCA"], active=0)
+    return radio_button_group
```

Comment:
I want this radio button group to be stretched to take up 50% of the total screen width (ie: it fits over the scatterplot nicely). I'll look for a good link on formatting button sizes (might be in buttons, or more likely in layouts).

Reply:
Hm, turns out that scaling the size to *half* of the screen may not be supported at all.

And it also appears that button groups don't respect the keyword arguments that do exist `sizing_mode`/`width`). Issue opened here: https://github.com/bokeh/bokeh/issues/8443